### DISCUSSION TITLE
M600 - "Unknown gcode_macro variable 'fan_speed"

### DISCUSSION THREAD

**Discussion by kaktuss77** [Guilouz/Klipper-Flsun-Speeder-Pad#147] [Q&A]
Hi all, 

I want to use M600 Macro to change color of my print at specific layer. but when I use it, i have this error. 

"Unknown gcode_macro variable 'fan_speed"

It's seam to appear also with the resume print macro

All files and version (klipper, etc) are up to date, 

anyone for help?

Thanks

Regards

**Comment by Guilouz** [comment]
Use latest config files.

**Reply by kaktuss77** [reply]
hi, oh updated from 3 days, sorry, have do mine 4days ago hahaha

Regards,

Merci pour ton super travail, au top vraiment.
